CONTRIBUTING.md

```diff
@@ -1,33 +1,51 @@
 # Contribution Guidelines
 
-## Branches
+Before publishing a PR, please test builds locally:
 
-* *master*:  images on the master branch are built monthly and with github action triggered by ERPNext release.
-* *develop*: images on this branch are built daily and when PR is merged into develop.
+- with docker-compose for production,
+- with and without nginx proxy,
+- with VSCode for testing environments (only for frappe/bench image).
 
-# Pull Requests
+On each PR that contains changes relevant to Docker builds, images are being built and tested in our CI (GitHub Actions).
+> :evergreen_tree: Please be considerate when pushing commits and opening PR for multiple branches, as the process of building images uses energy and contributes to global warming.
+> 
 
-Please **send all pull request exclusively to the *develop*** branch.
-When the PR are merged, the merge will trigger the image build automatically.
+## Lint
 
-Please test all PR as extensively as you can, considering that the software can be run in different modes:
-
-* with docker-compose for production
-* with or without Nginx proxy
-* with VScode for testing environments
+```shell
+shellcheck example.sh
+```
 
-Every once in a while (or with monthly release) develop will be merged into master.
+## Build
 
-There is Github Action is configured on ERPNext repository. Whenever there is a ERPNext release it will trigger a build on master branch of frappe_docker repo to generate images for released version.
+```shell
+# *...* — targets from docker-bake.hcl,
+# e.g. bench-build, frappe-socketio-develop or erpnext-nginx-stable.
+# Stable builds require GIT_BRANCH (e.g. v13.15.0), IMAGE_TAG (version-13), VERSION (13)
+# environment variables set.
+docker buildx bake -f docker-bake.hcl *...*
+```
 
-When a PR is sent, the images are built and all commands are tested.
+## Test
 
-If update or fixes to documentation are pushed use `[skip travis]` anywhere in commit message to skip travis.
+### Ping site
+Lightweight test that just checks if site will available after creation.
```

Comment:
Lightweight test that just checks if site will **be** available

Reply:
```suggestion
Lightweight test that just checks if site will be available after creation.
```